frontend/packages/component-library-styles/CONTRIBUTING.md

```diff
@@ -0,0 +1,153 @@
+# Contributing to Component Library Styles
+
+Thank you for considering contributing to the Component Library Styles! 🎉  
+We welcome all contributions — whether it’s a bug fix, new token, documentation improvement, or something else
+entirely.  
+This guide will help you get started quickly and ensure a smooth contribution process. To get the full picture of the
+project, please refer to the [README.md](./README.md) file.
+
+## Table of Contents
+
+- [🛣️ Contribution Roadmap](#-contribution-roadmap)
+- [🎨 How to Add a New Style or Token](#-how-to-add-a-new-style-or-token)
+- [🚦Production-ready Checklist](#production-ready-checklist)
+- [🎯 Coding Standards](#-coding-standards)
+- [🧪 How to Test Your Styles](#-how-to-test-your-styles)
+- [🔀 Commit Message Guidelines](#-commit-message-guidelines)
+- [📚 Need Help?](#-need-help)
+- [💡 Tips for a Smooth PR Review](#-tips-for-a-smooth-pr-review)
+- [✅ How to Get Your PR Approved Faster](#-how-to-get-your-pr-approved-faster)
+
+## 🛣️ Contribution Roadmap
+
+Here’s a quick guide to the contribution process:
+
+1. **(The most important step!) Create a thread** in the `#ask-design-system` Slack channel about your change.
+
+   - **Get approval** from @markabarrett and/or @moshebaricdo (design team).
+   - For technical details, ask @levadadenys.
+
+2. **Create a separate PR** with `@code-dot-org/component-library-styles` code changes.
+
+   - Make sure the PR follows the [Best Practices](./README.md/#best-practices).
+   - Include relevant updates to tokens, colors, typography, and styles.
+
+3. **Get it approved** by:
+
+   - ✅ @moshebaricdo and/or @markabarrett (design)
+   - ✅ @levadadenys (technical implementation)
+
+4. **Merge the PR** once approved.
+
+5. **Use it in your project → Celebrate → Collect tons of appreciation!** 🎉
+
+## 🎨 How to Add a New Style or Token
+
+Most of the styles come directly from the design system Figma file. Here's a list of what can be possibly added
+and how to do it:
+
+0. Check existing design tokens for duplicates or overlap to prevent redundancy.
+
+1. **Add the token** to the appropriate SCSS file:
+
+   - Fonts → `fonts.scss`
+   - Variables → `variables.scss`
+   - Mixins → `mixins.scss`
+
+2. Ensure the token follows existing naming conventions (e.g., `field-helper-section-white`).
+
+3. If adding a new style:
+
+   - Follow the existing structure and organization.
+
+4. **Test the token or style** across light/dark themes and RTL.
+
+5. Submit a PR following the [Contribution Roadmap](#-contribution-roadmap).
+
+## 🚦Production-ready Checklist:
+
+- Approved by the design team;
+- Defined in the appropriate SCSS file;
+- Tested in light and dark themes;
+- Verified in Storybook (if applicable).
```

Comment:
This should probably be a [pull request template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)

Reply:
It would be nice to automate it, but right now it's not automated, meaning I don't think we should remove it right now.
We've discussed it with @bethanyaconnor previously. I don't mind adding this to pr template, but only for related directories, I'm not sure if it's possible. Moreover, I'm not sure if everyone is checking the pr template constantly(same is true for docs, of course), meaning that either way, I think it's worth also keeping information in documentation so that people can read more about how everything works.